okapi/static/ocgpx/1/ocgpx.xsd

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xsd:schema
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+    xmlns="https://opencaching.pl/okapi/static/ocgpx/1"
```

Comment:
Perhaps it's safer to use xmlns in OKAPI's github namespace? See [here](https://github.com/erasmus-without-paper/ewp-specs-api-discovery/blob/v4.1.0/manifest-example.xml#L2) for example. GitHub URLs are highly unlikely to change, and we don't convey a bad impression that "this is for OCPL only".

I would even suggest to put this XSD outside of the OKAPI repository - we could make a separate repository solely for that single XSD (e.g. `https://github.com/opencaching/gpx-extension`), and introduce semantic versioning similar to the one I used in the example linked above (`stable-v1` branch). I have designed [tons of XSDs in this fashion](https://developers.erasmuswithoutpaper.eu/) in recent years and it seems to be working very well, so far.

Reply:
I am fine with `https://github.com/opencaching/gpx-extension`.

(Had in mind that there was a case when the Github path changed for such a document and OKAPI needed an adjustment. But maybe I am confusing this with something else.)

Reply:
Even if GitHub paths change, they still redirect properly. In other words, the value of `xmlns` will always work properly. This is not necessarily true for the value of `xsi:schemaLocation`, such as [this one](https://github.com/erasmus-without-paper/ewp-specs-api-discovery/blob/v4.1.0/manifest-example.xml#L11), but these links *can* break, so it's not a big deal (in fact, we don't even need to provide `xsi:schemaLocation`, if we don't want to).